eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -263,61 +263,8 @@ function New-DataPlanePackageFolder() {
     CreateOrUpdateAutorestConfigFile -autorestFilePath $file -namespace $namespace -inputfile "$inputfile" -readme "$readme" -autorestConfigYaml "$autorestConfigYaml"
     Update-CIYmlFile -ciFilePath $ciymlFilePath -artifact $namespace
   } else {
-    Write-Host "Path doesn't exist. create template."
-    if ($inputfile -eq "" -And $readme -eq "") {
-        Throw "[ERROR] Input file should not be empty. Please provide a valid input file or readme file."
-    }
-    dotnet new -i $sdkPath/sdk/template
-    Write-Host "Create project folder $projectFolder"
-    if (Test-Path -Path $projectFolder) {
-        Remove-Item -Path $projectFolder -ItemType Directory
-    }
-
-    Push-Location $serviceFolder
-    $namespaceArray = $namespace.Split(".")
-    if ( $namespaceArray.Count -lt 3) {
-        Throw "[ERROR] Invalid namespace name provided: $namespace. Please provide valid namespace."
-    }
-
-    $endIndex = $namespaceArray.Count - 2
-    $clientName = $namespaceArray[-1]
-    $groupName = $namespaceArray[1..$endIndex] -join "."
-    $dotnetNewCmd = "dotnet new azsdkdpg --name $namespace --clientName $clientName --groupName $groupName --serviceDirectory $service --force"
-    if ($inputfile -ne "") {
-        $dotnetNewCmd = $dotnetNewCmd + " --swagger '$inputfile'"
-    }
-    if ($securityScope -ne "") {
-        $dotnetNewCmd = $dotnetNewCmd + " --securityScopes $securityScope";
-    }
-
-    if ($securityHeaderName -ne "") {
-        $dotnetNewCmd = $dotnetNewCmd + " --securityHeaderName $securityHeaderName";
-    }
-
-    if (Test-Path -Path $ciymlFilePath) {
-        Write-Host "ci.yml already exists. update it to include the new serviceDirectory."
-        Update-CIYmlFile -ciFilePath $ciymlFilePath -artifact $namespace
-
-        $dotnetNewCmd = $dotnetNewCmd + " --includeCI false"
-    }
-    # dotnet new azsdkdpg --name $namespace --clientName $clientName --groupName $groupName --serviceDirectory $service --swagger $inputfile --securityScopes $securityScope --securityHeaderName $securityHeaderName --includeCI true --force
-    Write-Host "Invoke dotnet new command: $dotnetNewCmd"
-    Invoke-Expression $dotnetNewCmd
-
-    $file = (Join-Path $projectFolder "src" $AUTOREST_CONFIG_FILE)
-    Write-Host "Updating configuration file: $file"
-    CreateOrUpdateAutorestConfigFile -autorestFilePath $file -namespace $namespace -readme "$readme" -autorestConfigYaml "$autorestConfigYaml"
-    Pop-Location
-
-    $projFile = (Join-Path $projectFolder "src" "$namespace.csproj")
-    (Get-Content $projFile) -replace "<Version>*.*.*-*.*</Version>", "<Version>1.0.0-beta.1</Version>" | Set-Content $projFile
-    # dotnet sln
-    Push-Location $projectFolder
-    dotnet sln remove src/$namespace.csproj
-    dotnet sln add src/$namespace.csproj
-    dotnet sln remove tests/$namespace.Tests.csproj
-    dotnet sln add tests/$namespace.Tests.csproj
-    Pop-Location
+    Write-Error "Project directory doesn't exist."
```

Comment:
I assume this code won't be hit theoretically as new data plane services are required to onboard TypeSpec? Could you extend the message to include the Next Steps for the user?

Reply:
This code can be hit. When a swagger has existed, but it does not onboard .NET sdk before, and CI pipeline configures .NET SDK automation, it will hit this code.

And detail log to tell customer that we will not allowed to generate new sdk from swagger.